TheHiveV5/thehive/create_alert.py

```diff
@@ -14,12 +14,13 @@ def run(self, arguments: dict[str, Any]) -> Optional[OutputAlert]:
             organisation=self.module.configuration["organisation"],
         )
 
+        arg_sekoia_server = arguments["sekoia_base_url"]
         arg_alert = arguments["alert"]
 
         alert_type = f"{arg_alert['alert_type']['category']}/{arg_alert['alert_type']['value']}"
         if len(alert_type) > 32:
             alert_type = arg_alert["alert_type"]["category"][:32]  # limit to 32 char, max of thehive api
-        link = f"https://app.sekoia.io/operations/alerts/{arg_alert['short_id']}"
+        link = f"{arg_sekoia_server}/operations/alerts/{arg_alert['short_id']}"
```

Comment:
As the base url is supplied by the customer, to avoid issue in the construction of the url, I suggest using `posixpath.join` to compute the final url (like [this](https://github.com/SEKOIA-IO/automation-library/blob/a460e0cbfaf0ebfe10fcd76ab661cdb73000bfd0/Sekoia.io/sekoiaio/triggers/alerts.py#L102)).
```suggestion
        link = urljoin(arg_sekoia_server.rstrip("/"), "/operations/alerts/{arg_alert['short_id']}") # with `from posixpath import join as urljoin` in the import section of the file
```

The purpose is to avoid HTTP request on `https://app.sekoia.io//operations/alerts/11111` because the customer added a slash ('/') at the end of the base url.

Reply:
Normally it's done !